<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRooms</class>
 <widget class="QDialog" name="DialogRooms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Kat 1</string>
   </property>
   <widget class="QPushButton" name="pushbutton101">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>101</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton103">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>103</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton102">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>102</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Kat 2</string>
   </property>
   <widget class="QPushButton" name="pushbutton201">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>201</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton202">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>202(suite)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_okey">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>okey</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
